driver/src/contracts/stablex_contract.rs

```diff
@@ -270,6 +315,66 @@ pub mod tests {
         );
     }
 
+    #[test]
+    fn generic_parse_auction_data_test() {
```

Comment:
Are the bytes here different than the already existing unit tests? I would think that it might be added value to include a different bytes vector here.

Reply:
Hm... I like it the way it is. 
Here I am just testing that the bytes are correctly chunked into the elements.

Parsing the elements is tested in the other tests already sufficiently, in my opinion.

Reply:
Cool with me! I can't recall if we have tested this function with more than one Encoded Order?